pkg/api/router.go

```diff
@@ -60,6 +60,7 @@ func New(
 		r.Get("/*", quartoHandler.GetObject)
 		r.Post("/create", quartoHandler.Create)
 		r.Put("/update/{id}", quartoHandler.Update)
+		r.Put("/append/{id}", quartoHandler.Append)
```

Comment:
Burde vi ikke heller bruke request methods i stedet for navn på adressen?

Fra Mozilla:
> [POST](https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/POST)
>    The POST method submits an entity to the specified resource, often causing a change in state or side effects on the server.
> [PUT](https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/PUT)
>    The PUT method replaces all current representations of the target resource with the request payload.
> [PATCH](https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/PATCH)
>    The PATCH method applies partial modifications to a resource.

Reply:
Så patch?

Reply:
agree with Kyrre